### PR TITLE
feat(server): Create Migration

### PR DESCRIPTION
**Pull request by quannt-2733**
Database Diagram: https://link.sun-asterisk.vn/el0TjU

### CODE REVIEW

File: database/migrations/2020_06_26_064312_create_posts_table.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('posts', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('category_id');
+            $table->unsignedInteger('postable_id');
```
Comment (hieunv-0738) [inline database/migrations/2020_06_26_064312_create_posts_table.php:19]: $table->morphs('postable');

File: database/migrations/2020_06_26_070004_create_post_tags_table.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+class CreatePostTagsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('post_tags', function (Blueprint $table) {
```
Comment (tiennv-1572) [inline database/migrations/2020_06_26_070004_create_post_tags_table.php]:
```suggestion
        Schema::create('post_tag', function (Blueprint $table) {
```

File: database/migrations/2020_06_26_064328_create_tags_table.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('tags', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('tagable_id');
```
Comment (hieunv-0738) [inline database/migrations/2020_06_26_064328_create_tags_table.php:18]: $table->morphs('taggable');

File: database/migrations/2020_06_26_070004_create_post_tags_table.php (hunk 2 above)
Comment (hieunv-0738) [inline database/migrations/2020_06_26_070004_create_post_tags_table.php]: post_tag

**Review by tiennv-1572 — APPROVED** [review]

**Pull request merged**